database/queries.js

```diff
@@ -0,0 +1,15 @@
+const pgp = require( 'pg-promise' )()
+const db = pgp({ database: 'doitnow' })
+
+const allTasks = 'SELECT * from todos'
+const createTask = 'INSERT INTO todos (id, name) VALUES ($1, $2) RETURNING *'
+
+
+const Tasks = {
+  getAll: () => {
```

Comment:
Sure I can do that. Good idea! Another file tasks.js has been created with a modules.export